redis_enterprise/README.md

```diff
@@ -0,0 +1,65 @@
+# Agent Check: Redis Enterprise
+
+## Overview
+
+[Redis Enterprise][1] Software is a self-managed data platform that unlocks the full potential of Redis at enterprise scale. The speed you know and love, plus compliance, reliability, and unmatched resiliency for enterprise needs.
```

Comment:
The overview should include the below instead of Marketing copy: 
	1	What does your product do in 1-2 sentences
	2	What will customers get from this integration (the value of the integration & why it's helpful)
	3	What types of data sent and the value

Reply:
Marketing copy replaced, points 1-2 addressed, point 3 is answered in the Metrics section.

Reply:
This has been addressed